readthedocs/rtd_tests/tests/test_footer.py

```diff
@@ -418,7 +419,7 @@ class TestFooterPerformance(APITestCase):
 
     # The expected number of queries for generating the footer
     # This shouldn't increase unless we modify the footer API
-    EXPECTED_QUERIES = 14
+    EXPECTED_QUERIES = 13
```

Comment:
We have one query less bc we are not sending the `origin` parameter, so it defaults to false before doing a query for the feature flag.

Reply:
We should test this with an origin. I expect the `unresolve` method to actually add a few queries, which we should be careful about :/